install-template.sh

```diff
@@ -40,13 +68,31 @@ else
     exit 1
   fi
   install_nix
+  test_nix_installation || (echo "Failed to install nix, cancelling installation." && exit 1)
   echo nix is now installed:
   nix --version
 fi
-test_nix_installation
 
-echo TODO: testing binary cache...
-# is there a good way to test whether a binary cache is configured and available?
+echo
+echo BINARY CACHE CONFIGURATION
+echo ==========================
+
+if test_cache_configured; then
+  echo Hooray, the garnix cache is configured.
+else
+  echo For \'@@toolName@@\' to work well, it is recommended that you add the
+  echo garnix.io binary cache to your nix configuration. But the cache
+  echo configuration cannot be found.
+  printf "Should I add the garnix.io binary cache to /etc/nix/nix.conf now (requires sudo)? [y/n] "
```

Comment:
I think we should say somehow that we’ll otherwise abort the installation.

Reply:
I tweaked the whole section. It doesn't *explicitly say that it'll abort if you say no, but it implies it more heavily. Also it mirrors the prompt for the nix question now. In 573881fd. Is that good?

Reply:
Yup